cloud/aws/templates/aws_oidc/app.tf

```diff
@@ -96,8 +165,9 @@ module "td" {
 }
 
 module "ecs_fargate_service" {
-  source      = "cn-terraform/ecs-fargate-service/aws"
-  name_prefix = var.app_prefix
+  source        = "cn-terraform/ecs-fargate-service/aws"
+  name_prefix   = var.app_prefix
+  desired_count = 0 # TODO: set this to actual value
```

Comment:
Why are we setting this to 0 / not enabling it to actually bring up servers?  It seems a little confusing to do that, if all that it takes to actually get a server is to set this to non-zero.  It's not like anyone is using these scripts for anything but testing right now

Reply:
because if the task fails it will retry indefinitely, blowing through dockerhub quota for pulling images. 
We need to stop it from retrying indefinitely and then set this to a reasonable number.